src/components/SEO/SEO.jsx

```diff
@@ -1,6 +1,6 @@
 import React from "react";
 import urljoin from "url-join";
-import { useSiteMetadata } from "../../hooks/use-site-metadata";
+import { useSiteMetadata } from "../../hooks/useSiteMetadata";
```

Comment:
Not sure about our naming convention, but we mostly seem to use camel case for pages, components, etc.